content/podcast/2023/ep13_nextflow_10_years.md

```diff
@@ -0,0 +1,19 @@
+title=Nextflow turns 10!
+episode=13
+description=Phil Ewels, Paolo Di Tomasso and Evan Floden look back on 10 years of scientific workflows.
```

Comment:
spot the typo!

Reply:
I copied names from Twitter further down specifically to avoid typos 🙈 

Sorry! 😅 

```suggestion
description=Phil Ewels, Paolo Di Tommaso and Evan Floden look back on 10 years of scientific workflows.
```